### PR TITLE
Avoid using undefined value in generate_stateless_cookie_callback

### PR DESCRIPTION
**Pull request by bernd-edlinger**
##### Checklist

- [ ] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

**Review by InfoHunter — APPROVED** [review]

File: apps/s_cb.c [hunk 1]
```diff
@@ -819,7 +819,8 @@ int generate_stateless_cookie_callback(SSL *ssl, unsigned char *cookie,
 {
     unsigned int temp;
     int res = generate_cookie_callback(ssl, cookie, &temp);
-    *cookie_len = temp;
+    if (res != 0)
```
Comment (paulidale) [inline apps/s_cb.c:822]: Add the blank line above while you're here?

**Review by paulidale — APPROVED** [review]

File: apps/s_cb.c (hunk 1 above)
Comment (bernd-edlinger) [reply apps/s_cb.c:823]: sure.

**Comment by mattcaswell** [comment]
Looks like the same problem applies to master, so this fix should be applied there too.

**Comment by paulidale** [comment]
Yes for master.

**Comment by bernd-edlinger** [comment]
master solved this differently, and initialized temp = 0, so it's not undefined.

**Comment by bernd-edlinger** [comment]
do you still want me to add the if (res != 0) in master?

**Comment by mattcaswell** [comment]
Ah, sorry. That's me misreading the code. Ignore  me. The code actually looks a little different here in master anyway. 1.1.1 only is fine by me.

**Comment by paulidale** [comment]
I'd still be tempted to make this change in master too, just from the point of view of _don't change things on failure_.
I'm not all that fussed either way, though.

**Review by t8m — APPROVED** [review]

**Comment by t8m** [comment]
Merged to 1.1.1 branch. Thank you.

**Pull request closed without merging**